package messages;

message XBee_Relay_Resp {
	enum Code {
		SUCCESS = 1;
		ADDRESS_NOT_FOUND = 2;
		NETWORK_TIMEOUT = 3;
	}

	optional Code code = 1;
	optional int32 seq_no = 2;
}